inc/admin.php

```diff
@@ -59,6 +59,16 @@ public function inline_bootstrap_script() {
 		$bgclasses       = Optml_Lazyload_Replacer::get_lazyload_bg_classes();
 		$watcher_classes = Optml_Lazyload_Replacer::get_watcher_lz_classes();
 
+		$watchers = $this->settings->get_watchers();
+		$lazyload_bg_selectors = [];
+		foreach ( $watchers[ Optml_Settings::WATCHER_TYPE_LAZYLOAD ] as $selector ) {
+			$lazyload_bg_selectors[] = $selector;
+		}
+		foreach ( $bgclasses as $key ) {
```

Comment:
I'm not sure what this is used for? This was used for a different mechanism, in the `backgroundReplaceClasses` prop of the library. Is it doing the same thing?

Reply:
yes